transmart-batch/docs/data_formats/README.md

```diff
@@ -2,13 +2,17 @@
 
 ## For all data types
 * Parameters for all data types
-  * [study-params.md](study-params.md) - Parameters that are supported for all data types.
+  * [study-params.md](study-params.md) - Study level parameters regarding security, study ID and top node.
 * Metadata
   * [tags.md](tags.md) - Loading study, concept, patient metadata and links to source data per concept.
 
 ## Low-dimensional data
 * Clinical data
   * [clinical.md](clinical.md) - Loading numerical and categorical low-dimensional data from clinical, non-high throughput molecular profiling, derived imaging data, biobanking data or links to source data per patient.
+    * [word-mapping.md](word-mapping.md) - Redefine categorical values, without changing the source data.
+    * [ontology-mapping.md](ontology-mapping.md) - Map variables to ontology codes to create standardized cross-study concepts.
+    * [trial-visit-mapping.md](trial-visit-mapping.md) **(Not yet implemented)** - Provide a time unit and value for your trial visit labels.
```

Comment:
Indeed, we'll also need a way to add them to HD observations. These are observations now too. Wonder if @forus has a suggestion.